src/country_index.c

```diff
@@ -49,15 +49,19 @@
 #include "md_array.h"
 #include "hashtbl.h"
 
+extern int debug_flag;
 static hashfunc country_hashfunc;
 static hashkeycmp country_cmpfunc;
 
 #define MAX_ARRAY_SZ 65536
 static hashtbl *theHash = NULL;
 static int next_idx = 0;
 static GeoIP *geoip;
+static GeoIP *geoip6;
 static char *ipstr;
-static char unknown[20] = "__";
+static char *unknown = "??";
```

Comment:
Changing the format for unknown IP addresses will break backwards compatibility, is it important for you to know if its an unknown v4 or v6 ?